flepimop/gempyor_pkg/src/gempyor/__init__.py

```diff
@@ -1,4 +1,3 @@
 ## All functions are in minimal inference.
-
```

Comment:
@pearsonca I just remembered why I did this, it was to trigger the `gempyor-ci` as an example. I can remove this before merging.